src/components/listOfQuestions/index.test.js

```diff
@@ -0,0 +1,124 @@
+import { render, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { Question } from './index.js';
+import { validateQuestionDisplay } from './validations.js';
+import cuid from 'cuid';
+import { set } from 'date-fns';
+
+const questionFactory = ({
+	id = cuid(),
+	timestamp = set(new Date(), {
+		hours: 0,
+		minutes: 0,
+		seconds: 0,
+		milliseconds: 0,
+	}),
+	question = 'Placeholder question',
+	askee = 'Anonymous',
+	status = 'Unanswered',
+} = {}) => ({
+	id,
+	timestamp,
+	question,
+	askee,
+	status,
+});
+
+const noop = () => {};
+const renderQuestion = (
+	question,
+	{ onAccept = noop, onReject = noop } = {}
+) => {
+	render(
+		<Question onAccept={onAccept} onReject={onReject} question={question} />
+	);
+};
+
+test('When I render an accepted question, the question information is displayed', () => {
+	const question = questionFactory({ status: 'Accepted' });
+	renderQuestion(question), validateQuestionDisplay(question);
+});
+
+test('When I render a rejected question, the question information is displayed', () => {
+	const question = questionFactory({ status: 'Rejected' });
+	renderQuestion(question), validateQuestionDisplay(question);
+});
+
+test('When I render an unanswered question, the question information is displayed', () => {
+	const question = questionFactory();
+	renderQuestion(question), validateQuestionDisplay(question);
+});
+
+test('When I render an unanswered question, the question information is displayed and onAccept is called if I click the Accept button', () => {
+	const question = questionFactory();
+	const onAccept = jest.fn();
+	renderQuestion(question, { onAccept }), validateQuestionDisplay(question);
+
+	const acceptButton = screen.getByRole('button', { name: /accept/i });
+	userEvent.click(acceptButton);
```

Comment:
If you've already tested the action creators and selectors in your state model, and you've actually run the code and clicked the buttons, these tests might be redundant, or better performed in a functional test suite which hooks up with the real database so you can test end-to-end.